shogun-boot/src/main/java/de/terrestris/shogun/boot/service/ProjectionService.java

```diff
@@ -0,0 +1,86 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.terrestris.shogun.boot.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.log4j.Log4j2;
+import org.geotools.api.referencing.FactoryException;
+import org.geotools.api.referencing.operation.TransformException;
+import org.geotools.geometry.jts.JTS;
+import org.geotools.metadata.iso.extent.GeographicBoundingBoxImpl;
+import org.geotools.referencing.CRS;
+import org.geotools.referencing.proj.PROJFormattable;
+import org.geotools.referencing.proj.PROJFormatter;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.io.ParseException;
+import org.locationtech.jts.io.WKTReader;
+import org.locationtech.jts.io.WKTWriter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@Log4j2
+public class ProjectionService {
+
+    @Autowired
+    protected ObjectMapper objectMapper;
+
+    public JsonNode getProjectionDetails(String query) throws FactoryException {
+        var root = objectMapper.createObjectNode();
+        var system = CRS.decode(query);
+        root.put("status", "ok");
+        root.put("numberResult", 1);
+        var list = objectMapper.createArrayNode();
+        root.set("results", list);
+        var node = objectMapper.createObjectNode();
+        list.add(node);
+        var identifier = system.getIdentifiers().iterator().next();
+        node.put("authority", identifier.getAuthority().getTitle().toString());
+        node.put("code", identifier.getCode());
+        node.put("name", system.getName().toString());
+        PROJFormatter fmt = new PROJFormatter();
+        node.put("proj4", fmt.toPROJ((PROJFormattable) system));
+        var bbox = system.getDomainOfValidity();
+        var bboxList = objectMapper.createArrayNode();
+        for (var item : bbox.getGeographicElements()) {
+            var box = ((GeographicBoundingBoxImpl) item);
+            bboxList.add(box.getNorthBoundLatitude());
+            bboxList.add(box.getWestBoundLongitude());
+            bboxList.add(box.getSouthBoundLatitude());
+            bboxList.add(box.getEastBoundLongitude());
+        }
+        node.set("bbox", bboxList);
+        node.put("unit", system.getCoordinateSystem().getAxis(0).getUnit().getName().toLowerCase());
+        node.put("area", bbox.getDescription().toString());
+        node.put("wkt", system.toWKT());
+        return root;
+    }
```

Comment:
Absolutely minor, but for better readability I would prefer to create a POJO/DTO instead of a `JsonNode`.

Reply:
Done